app/views/shared/check_answers/_item.html.erb

```diff
@@ -8,7 +8,7 @@
     <%= question %>
   </dt>
   <!--  in-line styling due to issues around pdf generation in our production envs -->
-  <dd class="govuk-summary-list__value"<% if read_only %> style="text-align: right;"<% end %>>
+  <dd class="govuk-summary-list__value" style="<%= "text-align: right;" if read_only %>">
```

Comment:
Nice! I was looking at this earlier; it's a fix for the same problem which is causing https://github.com/ministryofjustice/laa-apply-for-legal-aid/pull/3485 to fail. I'll close that PR as you've fixed it here.